apps/system/manifest.webapp

```diff
@@ -147,6 +147,12 @@
       "rules": {
         "minimumAccessLevel": "certified"
       }
+    },
+    "sync-history": {
```

Comment:
If you want to do it with IAC, I would expose a connection called something like "places" (maybe investigate overlap with "search-results"), but preferably, I would avoid IAC and use the DataStore interface from the adapter, using shared code to make sure the data format of a Place is only defined in one place.